cibuildwheel/options.py

```diff
@@ -677,6 +678,9 @@ def build_options(self, identifier: str | None) -> BuildOptions:
             dependency_versions = self.reader.get("dependency-versions")
             test_command = self.reader.get("test-command", option_format=ListFormat(sep=" && "))
             before_test = self.reader.get("before-test", option_format=ListFormat(sep=" && "))
+            test_sources = self.reader.get(
+                "test-sources", option_format=ListFormat(sep=" ")
+            ).split()
```

Comment:
This doesn't handle filenames/directories with a space in it. Should we care about such things? I suppose it can't really hurt! so... the solution here would be to add an optional `quote` param to ListFormat, use it here like `ListFormat(sep=" ", quote=shlex.quote)`, and then use `shlex.split()` rather than `split()` here.

I'm happy to make that change if the above isn't clear!

Reply:
Makes sense - I'll add this to my todo list.

Reply:
I've just pushed an update that does shlex quote handling; not sure if there's any additional testing that you'd like to see - I couldn't find an obvious trove of explicit ListOption tests.

I'm still looking into the Pyodide test that failed on the [previous CI run](https://github.com/pypa/cibuildwheel/actions/runs/11790528257/job/33165456980).